packages/cfengine/fixup-for-termux.patch

```diff
@@ -0,0 +1,75 @@
+diff --git a/cf-agent/cf-agent.c b/cf-agent/cf-agent.c
+index 8c6341059..bd4056290 100644
+--- a/cf-agent/cf-agent.c
++++ b/cf-agent/cf-agent.c
+@@ -363,11 +363,11 @@ static void ConfigureBootstrap(GenericAgentConfig *config, const char *argument)
+         return;
+     }
+
+-    if(IsLoopbackAddress(argument))
+-    {
+-        Log(LOG_LEVEL_ERR, "Cannot bootstrap to a loopback address");
+-        DoCleanupAndExit(EXIT_FAILURE);
+-    }
++//    if(IsLoopbackAddress(argument))
```

Comment:
I will likely remove this code instead of commenting it out. Eventually we may add a `--allow-loopback-bootstrap` or similar if that makes sense.

Reply:
Maybe just ifdef it out for `__TERMUX__`?

Reply:
I actually got this changed in core a while back. So changed the diff to match that change which applies to any agent wanting to bootstrap to loopback interfaces.